### PR TITLE
Adds Boxer profession

### PR DESCRIPTION
**Pull request by 2birdie**
I copied and edited the Martial Artist profession (I hope that's okay to do. I'm sorry if it is not.) into a boxer profession. I did similar with the Martial Artist trait.

I don't know if I did this right, this is my first time submitting code stuff on github. I hope what I'm submitting is acceptable. Apologies if I did something wrong here.

### CODE REVIEW

File: data/json/professions.json [hunk 1]
```diff
@@ -3381,6 +3381,50 @@
...
+                "level": 2,
+                "name": "dodge"
+            }
+        ],
+        "items": {
+            "both": [
+                "shorts",
+                "gloves_wraps",
+				"boxing_gloves",
+				"headgear",
```
Comment (Coolthulhu) [inline data/json/professions.json]: You used tabs here. The project uses 4 spaces for indentation.
Notepad++ has an option to automatically replace tabs with spaces.

File: data/json/professions.json [hunk 2]
```diff
@@ -3381,6 +3381,50 @@
...
+                "boots"
+            ],
+            "male": [
+                "under_armor"
+                "under_armor_shorts"
+            ],
+            "female": [
+                "sports_bra",
+                "under_armor"
+                "under_armor_shorts"
```
Comment (Coolthulhu) [inline data/json/professions.json]: "male" and "female" blocks are for clothing that is sex-exclusive. They shouldn't share any entries. Move the identical entries to "both".

**Comment by 2birdie** [comment]
Thank you very much, Coolthulhu! I think I got it now. I just kinda copy and pasted the spacing used in other entries and fixed up the sex-exclusive clothing.

File: data/json/mutations.json [hunk 3]
```diff
@@ -4725,6 +4725,16 @@
         "profession": true
     },{
         "type" : "mutation",
+        "id" : "PROF_BOXER",
+        "name" : "Pugilist",
+        "points" : 0,
+        "description" : "You are experienced in the Sweet Science of boxing. You could've had a shot at a local boxing tournament, if the Cataclysm hadn't screwed that up.",
```
Comment (illi-kun) [inline data/json/mutations.json]: Please put double whitespace after a period.

File: data/json/professions.json [hunk 4]
```diff
@@ -3381,6 +3381,47 @@
         }
     },
     {
+                "type": "profession",
+        "ident": "boxer",
+        "name": "Boxer",
+        "description": "You were training for the fight of your life before the Cataclysm struck. Now you fight just to keep yourself alive.",
```
Comment (illi-kun) [inline data/json/professions.json]: and here too.

**Comment by 2birdie** [comment]
Thanks, illi-kun.

File: data/json/professions.json [hunk 5]
```diff
@@ -3381,6 +3381,47 @@
         }
     },
     {
+                "type": "profession",
```
Comment (Night-Pryanik) [inline data/json/professions.json]: Too many spaces here.

**Comment by 2birdie** [comment]
Thank you, Night-Pryanik.

**Comment by 2birdie** [comment]
Guess I should close this now. Huzzah! Thank you all for the help.

**Pull request merged**